ui/admin/.template-lintrc.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   plugins: ['ember-template-lint-plugin-prettier'],
   extends: ['octane', 'ember-template-lint-plugin-prettier:recommended'],
   rules: {
+    'simple-unless': false,
```

Comment:
What does this do?

Reply:
I was in a call with Cameron when he added this. Basically, the linter was getting upset because `{{#unless}}` was being used in conjunction with the `and` logical operator. The linter printed out an error saying that simple unless statements are recommended. So Cameron added that line in order to allow non-simple unless statements.

Reply:
Exactly what Lis said. I would prefer to use an inline rule exception but I did not see any docs for that. I plan to look into it but I didn't want this to be blocker for this work.

Reply:
Removed.